cmd/cephcsi.go

```diff
@@ -81,13 +81,13 @@ func init() {
 		"",
 		"list of Kubernetes node labels, that determines the topology"+
 			" domain the node belongs to, separated by ','")
-	flag.BoolVar(&conf.EnableReadAffinity, "enable-read-affinity", false, "enable read affinity")
 	flag.StringVar(
 		&conf.CrushLocationLabels,
 		"crush-location-labels",
 		"",
 		"list of Kubernetes node labels, that determines the"+
 			" CRUSH location the node belongs to, separated by ','")
+	flag.StringVar(&conf.ContainerOrchestrator, "container-orchestrator", "kubernetes", "container orchestrator")
```

Comment:
my suggestion was to add this extra flag but not to remove the `EnableReadAffinity` existing flag :)